examples/hello_world.rs

```diff
@@ -95,10 +95,10 @@ fn main() {
     };
 
     #[cfg(not(feature = "glsl-to-spirv"))]
-    let mut renderer = Renderer::new(&mut imgui, &device, &mut queue, sc_desc.format);
+    let mut renderer = Renderer::new(&mut imgui, &device, &mut queue, sc_desc.format, None, 1);
```

Comment:
I think we should make this a `RendererConfig` struct, now that it is so many arguments :)